build.gradle.kts

```diff
@@ -50,10 +50,16 @@ application {
 
 java {
     toolchain {
-        languageVersion.set(JavaLanguageVersion.of(8))
+        languageVersion.set(JavaLanguageVersion.of(11))
     }
 }
 
+tasks.compileJava {
+    javaCompiler.set(javaToolchains.compilerFor {
+        languageVersion.set(JavaLanguageVersion.of(8))
```

Comment:
Why are we compiling against jdk8?

Reply:
Because this is the lowest version supported by the mqtt-cli. It is sufficient to run it with only having java 8 installed.